docs/getting_started.md

```diff
@@ -0,0 +1,126 @@
+# Getting started
+
+At its heart Thanos provides a global query view, data backup, and access historical data as core features. All three are well separated and Thanos deployments can be adapted to make use of each of them individually. This supports using just a subset of its features as well as a gradual rollout that immediately provides some of its benefits.
+
+The following examples configure Thanos to work against a Google Cloud Storage bucket. However, any object storage (S3, HDFS, DigitalOcean Spaces, ...) can be used by using the equivalent flags to connect to the S3 API.
+
+## Requirements
+
+* One or more [Prometheus](https://prometheus.io) installations
+* An object storage bucket (optional)
+
+## Get Thanos!
+
+Thanos has no official releases yet. With a working installation of the Go toolchain, Thanos can be downloaded and built by running
+
+```
+go get github.com/improbable-eng/thanos/...
+```
+
+The `thanos` binary should now be in your `$PATH` and is the only thing required to deploy any of its components.
+
+## Sidecars
+
+Thanos integrates with existing Prometheus servers through a sidecar process, which runs on same machine/in the same pod as the Prometheus server itself. It only works with Prometheus instances of version 2.0 or higher.
+
+The sidecar is responsible for backing up data into an object storage bucket and providing querying access to the underlying Prometheus instance for other Thanos components.
+
+### Backups 
+
+The following configures the sidecar to backup data into a Google Cloud Storage bucket.
+
+```
+thanos sidecar \
+    --prometheus.url http://localhost:9090 \  # Prometheus's HTTP address
+    --tsdb.path /var/prometheus \             # Data directory of Prometheus
+    --gcs.bucket example-bucket \             # Bucket to upload data to
+```
+
+Rolling this out has little to zero impact on the running Prometheus instance. It is a good start to ensure you are backing up your data while figuring out the other pieces of Thanos.
+
+If you are not interested in backing up any data, the `--gcs.bucket` flag can simply be omitted.
```

Comment:
You could add a note about `s3` capabilities here, if you'd like. Just so folks know it's an option :)

Reply:
Yes!

Reply:
Ah, I actually already put a note on it at the beginning.